src/Symfony/Bridge/Doctrine/Tests/PropertyInfo/DoctrineExtractorTest.php

```diff
@@ -108,15 +111,24 @@ public function testTestGetPropertiesWithEmbedded()
     }
 
     /**
+     * @group legacy
+     *
      * @dataProvider legacyTypesProvider
      */
     public function testExtractLegacy(string $property, ?array $type = null)
     {
+        $this->expectDeprecation(sprintf('Since symfony/property-info 7.3: The "%s::getTypes()" method is deprecated, use "%s::getType()" instead.', DoctrineExtractor::class, DoctrineExtractor::class));
```

Comment:
Please replace all `sprintf()` calls by static strings in expected deprecations so that it's easier to read and spot any mistake